src/base/os.lua

```diff
@@ -610,6 +610,47 @@
 
 
 
+---
+-- Translate decorated command paths into their OS equivalents.
+---
+
+	function os.translateCommandsAndPaths(cmds, basedir, location, map)
+		local translatedBaseDir = path.getrelative(location, basedir)

```

Comment:
I tend to agree with the code here, just wanted to know your thoughts on having the caller perform this? Would there be any benefit in this function being "flexible"?